docs/server/features/archiving_blog_post.md

```diff
@@ -184,36 +184,36 @@ $ aws s3 ls s3://kurrentdb-cluster-123-archive/
 ```
 
 One strange thing you'll notice above is that the chunk file names in the archive always have a suffix of `.000001`.
-For example, `chunk-000000.000000` is stored as `chunk-000000.000001` which looks similar to a _scavenged_ chunk.
+For example, `chunk-000000.000000` is stored as `chunk-000000.000001`, which looks similar to a _scavenged_ chunk.
 
-The other strange thing you'll see is that if you have a _scavenged_ and _merged_ chunk that contains multiple logical chunks in it, it'll be stored as several chunk files in the archive.
-For example, _scavenged_ chunk `chunk-000012.000003` that contains chunks 12, 13 & 14 will be _unmerged_ and stored as `chunk-000012.000001`, `chunk-000013.000001` and `chunk-000014.000001` in the archive.
+The other strange thing you'll see is that if you have a _scavenged_ and _merged_ chunk that contains multiple logical chunks, it'll be stored as several chunk files in the archive.
+For example, _scavenged_ chunk `chunk-000012.000003`, which contains chunks 12, 13, and 14, will be _unmerged_ and stored in the archive as `chunk-000012.000001`, `chunk-000013.000001`, and `chunk-000014.000001`.
 
-This mechanism is in place to allow reading data from the archive quickly: Having a constant suffix (`.000001`) and one chunk file per logical chunk allows any node to quickly find the chunk file containing a certain _log position_ in the archive.
+This mechanism allows data to be read from the archive quickly. A constant suffix (`.000001`) and one chunk file per logical chunk allow any node to promptly find the chunk file containing a specific _log position_ in the archive.
```

Comment:
```suggestion
This mechanism allows data to be read from the archive quickly. A constant suffix (`.000001`) and one chunk file per logical chunk allow any node to quickly locate the chunk file containing a specific _log position_ in the archive.
```
i'm not sure if `promptly` is the right word, i've changed it to `quickly locate` but feel free to readjust it